app/assets/javascripts/admin/controllers/admin_pages_controller.js

```diff
@@ -0,0 +1,59 @@
+/**
+  This controller supports interface for creating pages in Discourse.
+
+  @class AdminPagesController
+  @extends Ember.Controller
+  @namespace Discourse
+  @module Discourse
+**/
+Discourse.AdminPagesController = Ember.Controller.extend({
```

Comment:
We're in the process of migrating all our JS code to ES6/ES2015. It would be appreciated if you could rename this file to `admin-pages.js.es6` and use `export default Em.Controller.extend({...})`